packages/bot/src/scratch/hooks/toolbox.js

```diff
@@ -75,136 +61,158 @@ Blockly.Toolbox.prototype.populate_ = function (newTree) {
     this.categoryMenu_.populate(newTree);
 };
 
-/**
- * deriv-bot: Show blocks for a specific category in flyout
- * @private
- */
-Blockly.Toolbox.prototype.showCategory_ = function (category_id) {
-    let flyout_content;
+Blockly.Toolbox.prototype.showSearch_ = function (search) {
```

Comment:
This function is called from the toolbox store, so it shouldn't be marked as a private function i.e. you can remove the underscore at the end.